app/concepts/api/v1/comments/operation/create.rb

```diff
@@ -0,0 +1,20 @@
+module Api::V1
+  module Comments::Operation
+    class Create < Trailblazer::Operation
+      step :model
+      step Policy::Guard(Lib::Policy::UserGuard.new), fail_fast: true
+      step Contract::Build(constant: Comments::Contract::Create)
+      step Contract::Validate(key: :comment), fail_fast: true
+      step Contract::Persist()
+      step :prepare_renderer
+
+      def model(ctx, params:, **)
+        ctx['model'] = Task.find(params[:task_id]).comments.new
```

Comment:
please move instantiation of the comment to the populator http://trailblazer.to/gems/reform/populator.html#populateifempty